internal/org/repository/eventsourcing/eventstore.go

```diff
@@ -132,6 +138,60 @@ func (es *OrgEventstore) ReactivateOrg(ctx context.Context, orgID string) (*org_
 	return model.OrgToModel(org), nil
 }
 
+func (es *OrgEventstore) OrgChanges(ctx context.Context, aggregateType es_models.AggregateType, id string, lastSequence uint64, limit uint64) (*org_model.OrgChanges, error) {
+	query := ChangesQuery(id, lastSequence, aggregateType)
```

Comment:
I would define AggregateType here, because in this func its always OrgAggregate
```suggestion
	query := ChangesQuery(id, lastSequence, model.OrgAggregate)
```